src/file/SDL_rwops.c

```diff
@@ -390,7 +391,10 @@ static Sint64 SDLCALL stdio_seek(SDL_RWops *context, Sint64 offset, int whence)
     }
 #endif
 
-    if (fseek(context->hidden.stdio.fp, (fseek_off_t)offset, stdiowhence) == 0) {
+    /* don't make a possibly-costly API call for the noop seek from SDL_RWtell */
+    is_noop = (whence == RW_SEEK_CUR && offset == 0);
```

Comment:
Same note I made for SDL3 about the parentheses:

```c
is_noop = (whence == RW_SEEK_CUR) && (offset == 0);
```

Otherwise this can be merged!

Reply:
> Please feel free to alter my branch, it exists only for the sake of this PR.

(Unless you'd rather I do it.)

Reply:
Thanks for the comment, this should be fixed now.